### PR TITLE
add sentence about how we connect coops and freelancers

### PR DESCRIPTION
**Pull request by King-Mob**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -69,8 +69,11 @@
 <body class="container">
 	<h1 class="mono">sepheo coop</h1>
 	<div id="about">
-		<p>Sepheo is the working title for a project which aims to become a mutual society for self-employed
-			professionals. We aspire to the <a href="https://en.wikipedia.org/wiki/Rochdale_Principles"
+		<p>SEPHEO (Self-Employed People Helping Each Other) Coop is a mutual society for self-employed
+			professionals. We make connections between coops and freelancers who want to work
```
Comment (wu-lee) [inline index.html:73]: > coops

I am wondering if we want to exclude non-coop organisations by phrasing it this way? 

I think it's be nice if we weren't exclusive, and could work with government, charities, or even limited companies. The coop connection could be implicit, or explicit, in that we are a coop and "principle six" applies.

**Review by wu-lee — COMMENTED** [review]
General thumbs up except for the comment about co-op exclusivity.

File: index.html [hunk 2]
```diff
@@ -70,8 +70,7 @@
 	<h1 class="mono">sepheo coop</h1>
 	<div id="about">
 		<p>SEPHEO (Self-Employed People Helping Each Other) Coop is a mutual society for self-employed
-			professionals. We make connections between coops and freelancers who want to work
-			in the coop economy.
+			professionals. We make it easy for freelancers to connect with coops and other aligned organisations.
```
Comment (wu-lee) [inline index.html:73]: Happy with this, although personally I don't think alignment is easy to check nor is it strictly required - we should just encourage it.

File: index.html (hunk 1 above)
Comment (wu-lee) [reply index.html]: Resolved in later commit/comment

File: index.html (hunk 2 above)
Comment (King-Mob) [reply index.html:73]: agreed

**Pull request merged**